geode-core/src/main/java/org/apache/geode/internal/cache/TXCommitMessage.java

```diff
@@ -2100,6 +2040,73 @@ public void run() {
     }
   }
 
+  HashSet getFarSiders() {
+    return farSiders;
+  }
+
+  DistributionManager getDm() {
+    return dm;
+  }
+
+  void doCommitProcessQuery(final InternalDistributedMember id) {
+    final TXCommitMessage message = TXCommitMessage.this;
```

Comment:
What class is this method "doCommitProcessQuery" on? I think it is TXCommitMessage.
So no need for this funny business of "TXCommitMessage.this" which was needed before because is was in an anonymous inner class.
Now everywhere you have "message" you can just have "this".
I already did this refactoring in the pull request that removes thread groups.
I would suggest for your fix not to refactor this chunk of code into its own method. Just delete the block of code from it that you want. That way it will be more clear what you are fixing and you will not have conflicts with the thread refactoring code.

Reply:
You are right that it is no longer in inner class. Cleaned up the code.